autogen/agentchat/conversable_agent.py

```diff
@@ -122,6 +123,9 @@ def __init__(
             default_auto_reply (str or dict): default auto reply when no code execution or llm-based reply is generated.
             description (str): a short description of the agent. This description is used by other agents
                 (e.g. the GroupChatManager) to decide when to call upon this agent. (Default: system_message)
+            messages (dict): the previous chat messages that this agent had in the past with other agens. Can be used to
```

Comment:
Since this is going to set the `chat_messages` attribute, we can name the argument `chat_messages`.